package.json

```diff
@@ -21,7 +21,7 @@
     "@guardian/commercial-core": "^5.4.0",
     "@guardian/consent-management-platform": "^10.11.1",
     "@guardian/core-web-vitals": "^2.0.1",
-    "@guardian/libs": "^10.1.1",
```

Comment:
> I'm not sure that this changes anything for DCR? As we're changing the root package.json instead of dotcom-rendering/package.json which already uses 13.1.0

This is Frontend not DCR ;)

As far as I'm aware we're building the commercial bundle in Frontend and then adding it to the page in DCR.